app/components/primer/dropdown.rb

```diff
@@ -0,0 +1,127 @@
+# frozen_string_literal: true
+
+module Primer
+  # `Dropdown` is a lightweight context menu for housing navigation and actions.
+  # They're great for instances where you don't need the full power (and code) of the select menu.
+  class Dropdown < Primer::Component
+    # Required trigger for the dropdown. Has the same arguments as <%= link_to_component(Primer::ButtonComponent) %>,
+    # but it is locked as a `summary` tag.
+    renders_one :summary, lambda { |**system_arguments, &block|
```

Comment:
What do you think about calling this `button`? `summary` refers to the underlying implementation, not the function.